batch/scripts/get_arguments.sh

```diff
@@ -100,6 +100,11 @@ do
       shift # past argument
       shift # past value
       ;;
+      -G|--export_to_gee)
+      EXPORT_TO_GEE="$2"
+      shift # past argument
+      shift # past value
+      ;;
```

Comment:
I think you just want 1 shift, and store something like "TRUE" (though I don't think the actual value matters). As is I think you'll be shifting past the next argument.

Reply:
> I think you just want 1 shift, and store something like "TRUE" (though I don't think the actual value matters). As is I think you'll be shifting past the next argument.

If it's confusing, see the --overwrite example below.

Reply:
oh good catch, fixed